src/js/localstorage.js

```diff
@@ -0,0 +1,187 @@
+//TESTING LOCALSTORAGE WHETHER IT WORKS OR EXIST 
+export function localStorageTest() {                 //to avoid name collisions
+    
+    try {
+        localStorage.setItem("test","test");
+        localStorage.removeItem("test");
+        console.log('Local Storage Fine')
```

Comment:
We don't need console to talk to us. Console logs are fine for debugging, but they should be removed then.

Reply:
Done.